src/_pytest/pytester.py

```diff
@@ -975,7 +976,7 @@ def _ensure_basetemp(self, args):
             args.append("--basetemp=%s" % self.tmpdir.dirpath("basetemp"))
         return args
 
-    def parseconfig(self, *args):
+    def parseconfig(self, *args: Union[str, py.path.local]) -> Config:
```

Comment:
Are you sure about the `py.path.local`? Seems a bit strange.

Reply:
Might have slipped in, and might also be there for tests / internal use only - will double-check it.

Reply:
I think this is the last thing to do before merging 😄

Reply:
I see that `_prepareconfig` specifically handled `py.path.local`, so it is probably indeed intended.